socs/agents/ifm_sbn246_flowmeter/agent.py

```diff
@@ -111,7 +111,13 @@ def acq(self, session, params=None):
             adr = "/iolinkmaster/port[{}]/iolinkdevice/pdin/getdata".format(dp)
             url = 'http://{}'.format(self.ip_address)
 
-            r = requests.post(url, json={"code": "request", "cid": -1, "adr": adr})
+            try:
+                r = requests.post(url, json={"code": "request", "cid": -1, "adr": adr})
+            except requests.exceptions.ConnectionError as e:
+                self.log.warn(f"Connection error occured: {e}")
+                pm.sleep()
```

Comment:
I don't think we need this sleep, you'll hit the `pm.sleep()` on line 108 next after the continue anyway. Other than that looks good!